src/app/routes/certRegRoutes.js

```diff
@@ -32,4 +38,36 @@ module.exports = function (app) {
         }
       }
     }))
-};
+
+    let courseId;
+    app.post(certRegServiceApi.reIssueCertificate,
+      bodyParser.json({ limit: '10mb' }),
+      permissionsHelper.checkPermission(),
```

Comment:
We should allow only admin can do these actions. Restrict for any other roles.

Reply:
You should add this in routes into whitelist api's list with the role checks

Reply:
Currently Dashboard can be seen only by MENTORS (that to if he has creator batches for that course) . SO Admin means => ("ORG_ADMIN","SYSTEM_ADMINISTRATION") or (the creator of the batch  (MENTOR)). ??